src/util.ts

```diff
@@ -22,12 +22,19 @@ const toBool = (value: unknown) => {
     }
   }
 
+  if (typeof value === "undefined") {
+    if (typeof defaultValue === "undefined") {
+      return true;
```

Comment:
I'm trying to understand why we return `true` if the value is `undefined`? Wouldn't we just want to call `Boolean` on the value?